test/runtests.jl

```diff
@@ -582,21 +587,3 @@ end;
     @test distance(h, SedgeDistanceDijkstra(2, 3, 3)) == 1
     @test distance(h, SedgeDistanceDijkstra(1, 3, 3)) == typemax(Int)
 end;
-
```

Comment:
same comment as above - why to remove?

Reply:
After removing the function, we also deleted the associated test.